### ISSUE TITLE
Not working with Nuxt CLI v3.0.0-rc.8

### ISSUE THREAD

**Issue by Defite** [cpsoinos/nuxt-svgo#49]
Options passed:

```js
svgoOptions: {
    svgo: false,
    defaultImport: "component",
    svgoConfig: {
      multipass: true,
      removeViewBox: false,
    },
  },
```

Usage in component:

```vue
<script setup lang="ts">
import IconBurger from "../assets/icons/burger.svg";

const props = defineProps({
  open: Boolean,
  onClick: Function,
});
</script>

<template>
  <button class="btn" @click="onClick">
    <IconBurger />
  </button>
</template>
```

When I build Nuxt, I get errors:

```
 WARN  Module nuxt-svgo is disabled due to incompatibility issues:                                                               13:54:29
 - [nuxt] Nuxt version `^3.0.0` is required but currently using `3.0.0-rc.8`

[nuxt] [request error] [unhandled] [500] Invalid value used as weak map key
  at WeakMap.set (<anonymous>)  
  at normalizePropsOptions (./node_modules/@vue/runtime-core/dist/runtime-core.cjs.js:3966:11)  
  at createComponentInstance (./node_modules/@vue/runtime-core/dist/runtime-core.cjs.js:6957:23)  
  at renderComponentVNode (./node_modules/@vue/server-renderer/dist/server-renderer.cjs.js:171:22)  
  at Module.ssrRenderComponent (./node_modules/@vue/server-renderer/dist/server-renderer.cjs.js:608:12)  
  at _sfc_ssrRender (./.nuxt/dist/server/server.mjs:6227:31)  
  at renderComponentSubTree (./node_modules/@vue/server-renderer/dist/server-renderer.cjs.js:253:13)  
  at renderComponentVNode (./node_modules/@vue/server-renderer/dist/server-renderer.cjs.js:188:16)  
  at Module.ssrRenderComponent (./node_modules/@vue/server-renderer/dist/server-renderer.cjs.js:608:12)  
  at _sfc_ssrRender (./.nuxt/dist/server/server.mjs:5969:31)
```

**Comment by cpsoinos** [comment]
Hi @Defite, thanks for flagging this! It looks like this issue was recognized by the nuxt core team in https://github.com/nuxt/framework/issues/6699, and was fixed in https://github.com/nuxt/framework/pull/7116/files. That's slated to be fixed in the upcoming 3.0.0-rc.9 release.

I implemented a workaround, as was pointed out in the issue above, which sets compatibility to `'^3.0.0-rc.0'`. In my testing this now works with `3.0.0-rc.8`.

Please upgrade to `v1.0.1` to take advantage of this fix!

**Comment by Defite** [comment]
> Hi @Defite, thanks for flagging this! It looks like this issue was recognized by the nuxt core team in [nuxt/framework#6699](https://github.com/nuxt/framework/issues/6699), and was fixed in https://github.com/nuxt/framework/pull/7116/files. That's slated to be fixed in the upcoming 3.0.0-rc.9 release.
> 
> I implemented a workaround, as was pointed out in the issue above, which sets compatibility to `'^3.0.0-rc.0'`. In my testing this now works with `3.0.0-rc.8`.
> 
> Please upgrade to `v1.0.1` to take advantage of this fix!

Worked for me, thank you!
